addons/website/static/src/snippets/s_carousel/000.js

```diff
@@ -0,0 +1,21 @@
+/** @odoo-module **/
+
+import publicWidget from "@web/legacy/js/public/public_widget";
+
+const Carousel = publicWidget.Widget.extend({
+    selector: ".s_carousel_wrapper",
+
+    /**
+     * @override
+     */
+    willStart() {
+        // TODO: remove in master
+        this.el.querySelector(".carousel").setAttribute("data-bs-ride", "carousel");
+
+        return this._super(...arguments);
+    },
+});
+
+publicWidget.registry.Carousel = Carousel;
```

Comment:
Remove this line & register public widget directly. 

`publicWidget.registry.Carousel = publicWidget.Widget.extend({`